src/man/sssd-passkey.5.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DocBook V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<reference>
+<title>SSSD Manual pages</title>
+<refentry>
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="include/upstream.xml" />
+
+    <refmeta>
+        <refentrytitle>sssd-passkey</refentrytitle>
+        <manvolnum>5</manvolnum>
+        <refmiscinfo class="manual">File Formats and Conventions</refmiscinfo>
+    </refmeta>
+
+    <refnamediv id='name'>
+        <refname>sssd-passkey</refname>
+        <refpurpose>SSSD passkey options</refpurpose>
+    </refnamediv>
+
+    <refsect1 id='description'>
+        <title>DESCRIPTION</title>
+        <para>
+            This manual page describes the specifics for configuration of
+            passkey for
+            <citerefentry>
+                <refentrytitle>sssd</refentrytitle>
+                <manvolnum>8</manvolnum>
+            </citerefentry>.
+            Refer to the <quote>FILE FORMAT</quote> section of the
+            <citerefentry>
+                <refentrytitle>sssd.conf</refentrytitle>
+                <manvolnum>5</manvolnum>
+            </citerefentry> manual page for detailed syntax information.</para>
+    </refsect1>
+
+    <refsect1 id='configuration-options'>
+        <title>CONFIGURATION OPTIONS</title>
+        <para>
+            <variablelist>
+                <varlistentry>
+                    <term>user_verification (boolean)</term>
+                    <listitem>
+                        <para>
+                            Enable or disable the user verification (i.e. PIN,
+                            fingerprint) during authentication.
+                        </para>
+                        <para>
+                            Three different actors come into play when deciding
+                            whether to request user verification: LDAP server,
+                            <citerefentry><refentrytitle>sssd.conf
+                            </refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                            option and the passkey device itself.
+                        </para>
+                        <para>
+                            For the IPA provider the server side settings have
+                            higher priority than <filename>sssd.conf
+                            </filename>. For every other provider the settings
+                            from <filename>sssd.conf</filename> have higher
+                            priority.
```

Comment:
I still think "priority" should be avoided here. If I understand it correctly in the IPA case only the server settings are used and for all other cases there are no server settings and only the settings from `sssd.conf` will be used. So there is no priority or ordering because there is always only a single source.

Reply:
It's a little bit more complicated since we have the offline authentication workflow. What do you think about the next proposal?
```
For online authentication with the IPA provider, where a Kerberos ticket is granted, server-side settings are applied. For other cases, such as offline authentication with the IPA provider or any authentication from another provider, the settings in sssd.conf are used.
```

Reply:
Hi,

I think the IPA provider should use the server-side settings even for the offline case if passkey authentication is available online. Otherwise it would be the same odd inconsistency we tried to avoid with `local_auth_policy = match` with respect to the available authentication methods.

Only if passkey authentication is not available online in the IPA case and is enable locally by setting `local_auth_policy = enable:passkey` the setting from `sssd.conf` can be used.

In a mixed case where users from a trusted AD forest have the required attributes for passkey authentication set and ``local_auth_policy = enable:passkey` must be set to allow passkey authentication for those users I think the server-side setting for IPA users should be applied as well.

bye,
Sumit

Reply:
So what about
```
If the IPA provider is used, passkey authentication is not available online and
local_auth_policy = enable:passkey is set, then the settings from sssd.conf are used.
For any other IPA provider case, server-side settings are applied. For other cases, such
as any authentication from another provider, the settings in sssd.conf are used.
```
?

And I fear I must have lost some case :disappointed_relieved:

Reply:
Hi,

I think it covers all with some aspects only implicit. But I would suggest to try to reorder the sentences so that the cases where sssd.conf is used are following each other. So maybe
```
If the IPA provider is used and online, Kerberos based passkey authentication
is available the server-side settings are applied for all passkey usages. For
other cases the settings in sssd.conf are used. This includes passkey
authentication with any other provider and with the IPA provider in case online,
Kerberos based passkey authentication is not available and local passkey
authentication is allowed by the option `local_auth_policy = enable:passkey`.
```

bye,
Sumit

Reply:
I changed some minor parts of the text, but overall it's almost the same. Thanks!